change/@ni-nimble-components-dcc5406d-9d6c-4a9a-91de-b11d16c290f8.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
As part of this spec we should add a section to the table column hld describing delegated events and update the current discussion which described the exact event handling behavior as TBD:

https://github.com/ni/nimble/blob/adae0d3ab801c7c87acf53cc48025d855465fd5c/packages/nimble-components/src/table/specs/table-columns-hld.md?plain=1#L350-L408

Reply:
Replaced that example with a brief section outlining the event delegation approach.